default/stringtables/en.txt

```diff
@@ -7935,57 +7943,113 @@ Launch Bay
 
 PC_FIGHTER_BAY_DESC
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Launched fighters join combat the round after being launched, returning once combat ends for that turn.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each weapon hit by the shield parts strength.
+
+Unlike a planet shield, a ship shield is a reflective shield and does not sustain any loss in strength from an attack. Ship shields are ineffective against [[encyclopedia FIGHTER_TECHS]].'''
+
 # Ship part classes
 PC_ARMOUR
 Armor
 
+PC_ARMOUR_DESC
+Armor parts expand the [[metertype METER_STRUCTURE]] of a ship, allowing a ship to sustain more damage before being destroyed.
+
 # Ship part classes
 PC_TROOPS
 Ground Troops
 
+PC_TROOPS_DESC
+'''Troop parts increase the [[metertype METER_TROOPS]] on-board, which can be used to invade enemy planets.
+
+To successfully invade a planet, the total number of [[metertype METER_TROOPS]] from all invading ships must exceed the total defending (ties are awarded to the defender).
+An invading ship is destroyed as part of the landing operation (any additional parts on the ship are lost as well).'''
+
 # Ship part classes
 PC_DETECTION
 Detection
 
+PC_DETECTION_DESC
+'''Detection parts increase the [[metertype METER_DETECTION]] for the ship they are equipped on.
+
+An objects [[metertype METER_STEALTH]] must fall within the empires [[encyclopedia DETECTION_TITLE]] value to be visible.'''
```

Comment:
An object's

An empire's detection strength must equal or exceed an object's stealth to detect the object.